test/capybara_extensions/assertions.rb

```diff
@@ -24,6 +24,6 @@ def assert_no_#{selector}(...)
   include CapybaraExtensions::Assertions
 
   def page
-    @page ||= Capybara.string(rendered)
+    @page ||= Capybara.string(rendered.to_s)
```

Comment:
If you pass `rendered.html`, you'll get a Nokogiri node.

Sorry to have broken this 😈

Reply:
@seanpdoyle by the way, it's a really cool change! I'm glad you did it, and it's cool to see `response.parsed_body` being a source of inspiration too 😄

I tried `rendered.html`! But I couldn't get it to work. I tried stuff like these:

```ruby
def page
  @page ||= Capybara.string(rendered.html)
end

def page
  rendered.html
end
```

And then just removing the `page` method altogether. If you have an idea of how we can simplify this, I'm all ears 😄